### ISSUE TITLE
19225074: Exceptions (-accessibilityColumnIndex sent to cells in NSTableCellView) when Accessibility Inspector is open

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#2488]
#### Description

Summary:
We’re seeing exceptions being thrown and a pop-up button unable to be interacted with when Accessibility Inspector is open.

Steps to Reproduce:
1. Download, unzip, and launch GitHub for Mac https://github-central.s3.amazonaws.com/mac/GitHub%20for%20Mac%20193.zip
2. Add a git repository with at least one commit to it (e.g. by dragging/dropping a repo onto the app).
3. Select the History tab.
4. Open Accessibility Inspector.
5. Open Console and filter it for GitHub.
6. In GitHub for Mac, click the little grey gear icon that’s alongside the details of the commit.

Expected Results:
I expected the menu that shows up normally when you click that icon to show up.

Actual Results:
Nada.

Regression:
As far as we’re aware this is new as of 10.10.

Notes:
In Console, there’s exceptions being thrown: `-[GHTintedPopUpButtonCell accessibilityColumnIndex]: unrecognized selector sent to instance 0xblargh`. We thought that meant it was down to that cell subclass, but you get the same exceptions from NSTextFieldCell when clicking/selecting other things in the table view.

The table view doesn’t have any custom AX properties set on it that I can see, nor does that row or cell view. The table view has been subclassed to accept first responder (so that e.g. diff text is selectable). The diff and diff header cells have some custom AX properties.

This is known to affect a small number of users of GitHub for Mac.
## 

Product Version: Mac OS X 10.10.1 (14B25)
Created: 2014-12-11T22:24:43.076968
Originated: 2014-12-11T17:24:00
Open Radar Link: http://www.openradar.me/19225074
